File: airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/SshPasswordPostgresSourceAcceptanceTest.java
/*
 * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.io.airbyte.integration_tests.sources;

import io.airbyte.integrations.base.ssh.SshTunnel;

public class SshPasswordPostgresSourceAcceptanceTest extends AbstractSshPostgresSourceAcceptanceTest {

  @Override
  public SshTunnel.TunnelMethod getTunnelMethod() {
    return SshTunnel.TunnelMethod.SSH_PASSWORD_AUTH;
  }

}

File: airbyte-integrations/connectors/source-postgres/src/test/kotlin/io/airbyte/integrations/source/postgres/legacy/SshPasswordPostgresSourceAcceptanceTest.kt
package io.airbyte.integrations.source.postgres.legacy

import io.airbyte.cdk.test.fixtures.legacy.SshTunnel

class SshPasswordPostgresSourceAcceptanceTest : AbstractSshPostgresSourceAcceptanceTest() {
    override val tunnelMethod: SshTunnel.TunnelMethod
        get() = SshTunnel.TunnelMethod.SSH_PASSWORD_AUTH
}
